src/lib.rs

```diff
@@ -86,6 +89,15 @@ async fn install() -> eyre::Result<()> {
 #[macro_export]
 macro_rules! include {
     () => {
-        ::std::include!(concat!(env!("OUT_DIR"), "/buffrs.rs",));
+        let output_directory = match env!("OUT_DIR") {
```

Comment:
lets not do this – cargo artifacts are only ever meant to be stored in the OUT_DIR

Reply:
@mara-schulke - as we've removed the rust generation...can this part be removed, or should it invoke Tonic somehow?

Reply:
We didnt remove rust generation!

Reply:
Hey, I was referring to this discussion point that you raised: https://github.com/helsing-ai/buffrs/pull/72#discussion_r1332754910 
Trying to determine if the call in lib.rs should instead pass in Generator::Tonic instead of the old Language::Rust?

Reply:
This is resolved now, lib.rs can still use Tonic to generate code